src/editable-heading/editable-heading.tsx

```diff
@@ -153,16 +180,35 @@ export const EditableHeading = (props: EditableHeadingProps) => {
                 disabled={isShortcutsDisabled}
               />
 
-              <input
-                className={inputClasses}
-                value={children}
-                autoFocus={autoFocus}
-                data-test={dataTest}
-                disabled={isSaving}
-                {...restProps}
-                onFocus={onInputFocus}
-                onBlur={onInputBlur}
-              />
+              {!multilineInput
+                ? (
+                  <input
+                    className={inputClasses}
+                    value={children}
+                    autoFocus={autoFocus}
+                    data-test={dataTest}
+                    disabled={isSaving}
+                    {...restProps}
+                    onFocus={onInputFocus}
+                    onBlur={onInputBlur}
+                  />
+                )
+                : (
+                  <div ref={textAreaWrapperRef} className={styles.textareaWrapper}>
```

Comment:
Is the `div` wrapper absolutely required here? If you maybe can share what is it's purpose, we can understand it better

Reply:
Yes, the answer is in the previous comment